torch/distributed/device_mesh.py

```diff
@@ -538,132 +531,93 @@ def _setup_world_group_and_device(self):
 
             return _get_default_group()
 
-        def _init_process_groups(
-            self,
-            backend_override: tuple[
-                tuple[Optional[str], Optional[C10dBackend.Options]], ...
-            ],
-        ):
-            # group_name associated with each mesh dimension, each
-            # mesh dimension should have one sub-group per rank
-            #
-            dim_group_names: list[str] = []
-            default_group = _get_default_group()
+        @staticmethod
+        def _from_layouts(
+            device_type: str,
+            layouts: tuple[_Layout, ...],
+            cur_rank: int,
+            *,
+            dim_names: Optional[Union[str, tuple[str, ...]]] = None,
+        ) -> "DeviceMesh":
+            """
+            Creates a DeviceMesh from an existing backend and layouts. This will create a new device mesh
+            from a same backend universe or transformed from (e.g, flatten, etc.). Although CuTe layout makes
```

Comment:
from a same backend universe or transformed from - this is both not grammatical and introduces terms that aren't well defined (backend universe =?)